general-information.md

```diff
@@ -178,7 +178,7 @@ Stops describe vehicle trip start and end locations in a pre-designated physical
 | stop_id                | UUID                                                        | Required          | Unique ID for stop                                                                           |
 | name              | String                                                      | Required          | Name of stop                                                                                 |
 | last_reported          | Timestamp                                                   | Required          | Date/Time that the stop was last updated                                                     |
-| location               | GeoJSON [Point Feature](provider/README.md#geographic-data) | Required          | Location of the Stop                                                                         |
+| location               | GeoJSON [Point or Polygon Feature](provider/README.md#geographic-data) | Required          | Location of the Stop. If Point is used then assume a 20 meter buffer.                                                                         |
```

Comment:
Reference should to be updated to point to Geographic Data section in this document, as it was moved from Provider readme.

Reply:
Fixed in the latest commit thanks.